appliances/backup_and_restore.md

```diff
@@ -35,17 +35,22 @@ Before adding the cron job it would be best practice to mount an external storag
 # Restore
 Restoring a backup of the appliance will be done using SSH or the console.
 
-1. Download a backup TGZ and place it in `/`
-2. On the command line:
+1. Set the hostname on the new appliance.  On the command line:
+   ```bash
+     hostnamectl hostname your-new-hostname.example.com
+     echo "your.ip.address your-new-hostname.example.com your-new-hostname" >> /etc/hosts
+   ```
+2. Download a backup TGZ and place it in `/`
+3. On the command line:
    ```bash
      cd /
-     tar -zxf /backup
-     rm -f /backup # if desired to save space
+     tar -zxf /backup.tgz
+     rm -f /backup.tgz # if desired to save space
    ```
-3. Launch `appliance_console`
-   1. If this is a brand new appliance and the database has not yet been initialized, select `Configure Application`. If this is an existing database appliance, this step can be skipped.
```

Comment:
There are 2 possible scenarios 
1. restoring from a backup to the same appliance  - in this case, there is no need to create a database 
2. restoring from a backup to a new appliance - in this case, it is necessary to create a database as per the steps below

Reply:
1) True.  The problem is configuring the database on a brand new appliance does more than create a database.  It takes the empty disk, sets it up with physical and logical volumes and initializes postgres including the data directory but also the configuration and permissions.  I probably missed other things it's doing.  So, we can't really restore a database directly on a "new" appliance before all of these things are done.  That's why this section is here.  If we can move to asking for the tgz file, from the backup step, we can detect if there's no v2_key for the backup and warn the user.  That way, we can ensure the v2_key is copied over before we try to initialize postgresql and do the restore.

I think for now, we have to do the simplest things we can do to make this more clear.